hw/dv/tools/dvsim/sim.mk

```
@@ -73,7 +73,7 @@ ifneq (${sw_images},)
 			exit 1; \
 		fi; \
 		prebuilt_path=`echo $$sw_image | cut -d: -f 1`; \
-		bazel_label=`echo $$sw_image | cut -d: -f 1-2`; \
+		bazel_label="`echo $$sw_image | cut -d: -f 1-2`_sim_dv"; \
```

```suggestion
		bazel_label="`echo $$sw_image | cut -d: -f 1-2`_$$sw_build_device"; \
```

In DV, we have support to run any image type, not just `sim_dv` ones. This is instrumented in the hjson with `sw_build_device`, which can be overridden on a per test basis.

---

ah, right. thanks.